configuration-examples/slotting-facet-options/README.md

```diff
@@ -0,0 +1,52 @@
+# Slotting Facet Options
```

Comment:
Would it make more sense to change this to "Globally Slotting Facet Options" or something similar?